### PR TITLE
add example to show the calculations of expectation values of operators

### PR DESCRIPTION
**Pull request by shenmidelin**

### CODE REVIEW

File: examples/more/ED/6orb/run_ed.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    J2 = np.dot(Jxyz[0], Jxyz[0]) + np.dot(Jxyz[1], Jxyz[1]) + np.dot(Jxyz[2], Jxyz[2])
+    Jz = Jxyz[2]
+
+    # We can also calculate the expectation values of the occupancy number of
+    # t2g orbitals, dxz, dyz, dxy, :math:`\\hat{d}^{\\dagger}_{\\alpha}\\hat{d}_{\\alpha}`
+    # First, write their matrice in the real harmonics basis
+    # |dxz,up>, |dxz,dn>, |dyz,up>, |dyz,dn>, |dxy,up>, |dxy,dn>
+    # In this basis, they take simple form, only the diagonal terms have element 1
+    nd_oper = np.zeros((norb, norb, norb), dtype=np.complex)
+    nd_oper[0, 0, 0] = 1
```
Comment (mpmdean) [inline examples/more/ED/6orb/run_ed.py:88]: Maybe
idx = np.arange(norb)
nd_oper[idx, idx, idx] = 1

File: examples/more/ED/6orb/run_ed.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python
+# usage: python run_ed.py
+
+import numpy as np
+import scipy
+import edrixs
+
+
+def do_ed():
```
Comment (mpmdean) [inline examples/more/ED/6orb/run_ed.py:9]: I think it's better to remove the function and make the whole thing a simple script. Th function makes it harder for nonexperts to interrogate the values.

File: examples/more/ED/6orb/run_ed.py (hunk 1 above)
Comment (shenmidelin) [reply examples/more/ED/6orb/run_ed.py:88]: Yes, that will be much simpler and definitely good style in the main codes. However, I am afraid that some users who are not good at Python language will get lost at that point, so, in the examples, I think we should keep the codes as simple as possible for users.

File: examples/more/ED/6orb/run_ed.py (hunk 2 above)
Comment (shenmidelin) [reply examples/more/ED/6orb/run_ed.py:9]: OK, good idea.

File: examples/more/ED/6orb/run_ed.py (hunk 1 above)
Comment (mpmdean) [reply examples/more/ED/6orb/run_ed.py:88]: I don't mind keeping it as it if you prefer.

**Pull request merged**